src/jdk.compiler/share/classes/com/sun/tools/javac/util/Log.java

```diff
@@ -261,12 +262,27 @@ protected Log(Context context) {
      * @param context the context in which to find writers to use
      * @return a map of writers
      */
+    private final static Charset nativeCharset;
```

Comment:
Inserting this static initializer in the middle of a method, between its javadoc and impl, is odd.